docs-ref-conceptual/manage-azure-subscriptions-azure-cli.md

```diff
@@ -102,12 +120,27 @@ az account set --subscription "My Demos"
 
 # change the active subscription using the subscription ID
 az account set --subscription "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
+```
+
+---
```

Comment:
This line isn't really needed.  I took it out of the Contributor's Guide as well.

Reply:
Done

Reply:
@v-abdriaz , the extra `---` is still there so I have edited your branch and removed it.